src/services/blockExplorer.js

```diff
@@ -265,12 +239,31 @@ function getETHTransactions(
   startblock: number,
   endblock: number,
 ): Promise<any> {
-  return callApi({
+  const apiSubdomain: ?BlockExplorerAPISubdomain = ENDPOINT_NAMES_BY_NETWORK_ID[networkId]
+
+  if (!apiSubdomain) {
+    throw new Error('Block Explorer Private Network Error')
+  }
+
+  const apiEnpoint: string = `${EXPLORER_API}/v1/${apiSubdomain}/${owner}/transactions`
+
+  const params: BlockExplorerAPIParams = {
     endblock,
     startblock,
     sort: 'desc',
     action: 'txlist',
-  }, networkId, owner)
+  }
+
+  const queryParams: string = Object
```

Comment:
There is a good lib called [qs](https://www.npmjs.com/package/qs) for that. It also handles edge cases and has great test coverage. Take a look.